document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoUtils.java

```diff
@@ -145,6 +150,20 @@ public static ReturnDocument getReturnDocument(final ReturnDocumentType returnDo
                     String.format("Unhandled return document type: %s", returnDocumentType)));
   }
 
+  public static boolean isFunctionExpressionSelectionWithGroupBy(
```

Comment:
Utils is only for very specific and common things. Can we move this method out of the utils?

Reply:
this is used at 2 places at `MongoGroupTypeExpressionParser` and `MongoSelectTypeExpressionParser`
so we'll need to define it at some common place, right?

Reply:
It makes sense to put them in a common place, but doesn't feel like "Utils" is the right place. Either we could have a shared class (non-static) or have the definition in one place and access it in another.